tasks/Task.cpp

```diff
@@ -845,6 +845,12 @@ void Task::exposureConfiguration(Arena::IDevice& device)
         }
     }
     else if (image_config.exposure_auto == ExposureAuto::EXPOSURE_AUTO_OFF) {
+        double exposure_time_seconds = image_config.exposure_time.toSeconds();
+        double seconds_per_frame = 1 / image_config.frame_rate;
+        if (exposure_time_seconds > seconds_per_frame - 0.0001) {
+            throw std::runtime_error("EXPOSURE TIME SURPASSES FPS");
```

Comment:
While the all caps

"surpasses" -> "is bigger than the acquisition period needed by the configured frame_rate"